### PR TITLE
Fix killing 12

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: library/types/src/modules/String.rb [hunk 1]
```diff
@@ -492,11 +492,11 @@ def ParseOptions(options, parameters)
...
-                "b"  => "", # backspace
+                "b"  => "\b", # backspace
                 "e"  => "e", # escape
-                "f"  => "", # FF
+                "f"  => "\f", # FF
                 "n"  => "\n", # NL
-                "r"  => " ", # CR
+                "r"  => "\b", # CR
                 "t"  => "\t", # tab
                 "v"  => "v", # vertical tab
```
Comment (lslezak) [inline library/types/src/modules/String.rb]: Interestingly, the backslash was lost before `v`. It looks like a different YCP-killer bug.

See the original code before the conversion: https://github.com/yast/yast-yast2/blob/good-bye-ycp/library/types/src/String.ycp#L518

Comment (jreidinger) [inline library/types/src/modules/String.rb]: thanks for catching, Thankfully it is only place here

**Comment by lslezak** [comment]
LGTM, thanks! :+1:

**Comment by mvidner** [comment]
@jreidinger @lslezak it seems you have replaced carriage return with a
backspace.
Dne 2015 1 27 10:53 "Josef Reidinger" <notifications@github.com> napsal(a):

**Comment by jreidinger** [comment]
@mvidner where?

**Pull request merged**